Add error log statement if an error occurs while listing from resource group

louisfelix opened the pull request:
The code was silently failing if a problem occurred while connecting
to the API (e.g., a permission problem)

Commit:
Add error log statement if an error occurs while listing from resource group

The code was silently failing if a problem occured while connecting
to the API (e.g., a permission problem)

vincentcreusotfx commented on main.go:
```diff
@@ -125,6 +125,8 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 
 		resources, err := ac.listFromResourceGroup(target.ResourceGroup, target.ResourceTypes)
 		if err != nil {
+			log.Printf("Failed to get resources for resource group %s and resource types %s: %v",
```
Is it a debug or a fatal ? I guess you're right just checking

louisfelix replied:
I used same log statement as other errors logging in the project.

Commit:
Add documentation to help new users (#46)

* Indent headers to sub-headers for better reading experience

* Add an "Install" sub-heading to the README

* Adding "Usage" sub-heading to the README

* Add some details to the README configuration sections

  * Change "./Azure-metrics-exporter" to "./azure-metrics-exporter" to
reflect default binary name
  * Add an "Exporter configuration" sub-heading to re-group corresponding
section and inform on the default "azure.yml"
  * Add an "Azure account requirements" under "Exporter Configuration"
  * Give the "Prometheus configuration" it's own section to avoid mixed
exporter config and prometheus config

* In README, remove the "Work in progress"

* Reference --help instead of pasting result

Also use same markdown for binary examples.
